File: algo-sum-pairs/python/sum_pairs.py
def sum_pairs(num_list, sum_value):
    output_list = []
    for value in num_list:
        for next_values in num_list:
            if (value + next_values == sum_value and value <= next_values):
                output_list.append([value, next_values])
    if len(output_list) == 0:
        return "unable to find pairs"
    elif len(output_list) == 1:
        return output_list[0]
    else:
        return output_list
        
        
        
        
        
print(sum_pairs([1, 2, 3, 4, 5], 7))

File: algo-sum-pairs/js/sumPairs.js
function sumPairs(numArr, sumValue) {
    outputArr = [];
    for (let i = 0; i < numArr.length; i++) {
        for (let y = i; y < numArr.length; y++) {
            if (numArr[i] + numArr[y] == sumValue) {
                outputArr.push([numArr[i], numArr[y]])
            }
        }
    }
    if (outputArr.length == 0) {
        return "unable to find pairs";
    } else if (outputArr.length == 1) {
        return outputArr[0];   
    }
    else {
        return outputArr;
    }
}

module.exports = sumPairs;